### PR TITLE
Use electron-builder instead of electron-packager?

### PR DESCRIPTION
**Pull request by mzwing**
Electron-builder is much more useful than electron-packager. For example, it can pack deb, appimage and so on.

### CODE REVIEW

**Comment by mzwing** [comment]
Also, maybe later we can use autoupdater.

**Comment by mzwing** [comment]
You can view it on [my fork](mzwing/bbg)

**Review by baiyang-lzy — APPROVED** [review]
awesome work!

**Comment by Misaka13514** [comment]
Have you tested the built package? @mzwing  We found a flickering problem with the built package.

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,4 @@ bbg-mas-x64
 bbg-darwin-x64
 bbg-win32-x64
 .vs
-dist
+dist
```
Comment (Misaka13514) [inline .gitignore:7]: It would be better if a new line (`\n`) was added at the end of the file.

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node-linker=hoisted
```
Comment (Misaka13514) [inline .npmrc:1]: Don’t [use](https://github.com/electron-userland/electron-builder/issues/683#issuecomment-241214075) [npm](http://electron.atom.io/docs/tutorial/using-native-node-modules/#using-npm) (neither .npmrc) for configuring electron headers. Use `electron-builder node-gyp-rebuild` instead.

`https://www.electron.build/configuration/configuration#configuration`

File: electron-builder.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
+{
+  "productName": "Baiyang-lzy's Blog Generator",
+  "appId": "com.github.baiyang-lzy.bbg",
+  "asar": true,
+  "files": [
+    "dist/electron/**/*",
+    "./main.js",
+    "App/*"
```
Comment (Misaka13514) [inline electron-builder.json:8]: `App/*` doesn't match all files under `App/`, such as `App/blog_source`.

https://www.electron.build/file-patterns#multiple-glob-patterns

File: .gitignore (hunk 1 above)
Comment (Misaka13514) [reply .gitignore:7]: Resolved in [5ebb156](https://github.com/baiyang-lzy/bbg/commit/5ebb156fba0a917a701e682e6b0c00aabb3193a7).

File: .npmrc (hunk 2 above)
Comment (Misaka13514) [reply .npmrc:1]: Resolved in [5ebb156](https://github.com/baiyang-lzy/bbg/commit/5ebb156fba0a917a701e682e6b0c00aabb3193a7).

File: electron-builder.json (hunk 3 above)
Comment (Misaka13514) [reply electron-builder.json:8]: Resolved in [5ebb156](https://github.com/baiyang-lzy/bbg/commit/5ebb156fba0a917a701e682e6b0c00aabb3193a7).

**Pull request merged**